diagnostic.rb

```diff
@@ -10,7 +12,12 @@
 end
 
 # question 4:
+Licensee.where hair_color: %w(Hazel), eye_color: %(Black Brown)
 
 # question 5:
+Licensee.where surname: %w(Escobar Whaley) given_name: %w(Marilynn Chris)
+rails g migration ChangeHeightToOneInchShorter height:height-1
```

Comment:
Whoa who whoa. You don't write a migration to change values in the table - you write migrations to change the _structure_ of a table. 

You're overthinking this. Once you've grabbed the rows using `.where`, you can just treat the output like an array of Licensee objects. These objects have a property called `.height` already, so you can just write `licensee.height -= 1`.